content/millennium/r4/proprietary-codes-and-systems.md

```diff
@@ -100,6 +100,7 @@ The Encounter Resource supports proprietary codes for:
 * Encounter.hospitalization.dietPreference codes are maintained in [Code Set 18 Diet](#code-set-18-diet)
 * Encounter.hospitalization.specialCourtesy codes are maintained in [Code Set 67 VIP](#code-set-67-vip) and [Code Set 16 Courtesy](#code-set-16-courtesy)
 * Encounter.hospitalization.dischargeDisposition codes are maintained in [Code Set 19 Discharge Disposition](#code-set-19-discharge-disposition)
+* Custom Attribute extensions on Encounter may contain Custom Attribute Value codes. These codes may be from [any Code Set](#list-of-code-sets)
```

Comment:
let's mention the type here instead.. Codeable Concept

Reply:
[How is this wording?](https://github.com/cerner/fhir.cerner.com/pull/438/commits/75fa1b49d349b619f0f4793753c1af3d906790a4)

Reply:
[Worked with Kathy to update wording](https://github.com/cerner/fhir.cerner.com/pull/438/files#diff-6fdcb00b5b1b6edd51b687794bea6b95R103)